packages/block-editor/src/utils/get-font-styles-and-weights.js

```diff
@@ -105,7 +108,11 @@ export function getFontStylesAndWeights( fontFamilyFaces ) {
 		}
 
 		// Format font style and weight values.
-		const fontWeight = formatFontWeight( face.fontWeight );
+		const fontWeight = formatFontWeight(
+			'number' === typeof face.fontWeight
+				? face.fontWeight.toString()
```

Comment:
Could we coerce all values to string at the top?

I'm thinking, if we make sure all items in the `fontWeights` array are string here, at the source, the `toString()` coercions above in packages/block-editor/src/components/global-styles/typography-utils.js

Might not be required.

Reply:
This is a very good idea @ramonjd ! I actually tried to implement it. But later I stopped, because I was afraid of adding an regression because we're modifying the value fetched from the JSON. So for now, I'm just avoiding doing it in this PR. But WDYT about creating an issue where we properly test all implications of and modify the values at the top?

Thank you!

Reply:
Fair enough - probably a premature optimization. Good to have the bug fixed first 😄